recipes/dgl/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "dgl" %}
+{% set version = "1.1.0" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/dgl
+  git_rev: {{ version }}
+  patches:
+    - conda-build.patch
+
+build:
+  skip: true  # [(not linux) or (cuda_compiler_version in (undefined, 'None'))]
+  number: 0
+  rpaths:
+    - lib/
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64]
```

```suggestion
    - '*/libcuda.*'  # [linux64]
    - "$RPATH/libtorch_cpu.so"
    - "$RPATH/libc10.so"
    - "$RPATH/libtorch.so"
```

If PyTorch added to build requirments